### DISCUSSION TITLE
How to build?

### DISCUSSION THREAD

**Discussion by ciarancoffey** [kyujin-cho/pixel-volte-patch#62] [Q&A]
This is a basic question. I've not built android apps before, or if I did it was over 10 years ago.
How do I build this?
I'm making assumptions android studio should be used.
I've installed the Android 13 SDK.
I get import issue for 
```
    import android.telephony.TelephonyFrameworkInitializer
    import com.android.internal.telephony.ICarrierConfigLoader
    import com.android.internal.telephony.IPhoneSubInfo
    import com.android.internal.telephony.ISub
    import com.android.internal.telephony.ITelephony
```

    file:///home/<username>/repos/pixel-volte-patch/app/src/main/java/dev/bluehouse/enablevolte/Moder.kt:10:26 Unresolved reference: TelephonyFrameworkInitializer
    
etc. 
Do I need to have additional SDK's installed?

**Comment by kyujin-cho** [comment]
Hi! Please check [here](https://github.com/kyujin-cho/pixel-volte-patch/issues/3#issuecomment-1421114938) to build the application.
